test/cli/test_cli_functions.sh

```diff
@@ -668,3 +668,37 @@ function check_backingstore {
     echo_time "💬  Deleting bucket testbucket"
     test_noobaa bucket delete "testbucket"
 }
+
+function check_dbdump {
+    # Generate db dump
+    echo_time "💬  Generaiting db dump"
+    test_noobaa db-dump --dir /tmp
+
+    # Check whether dump was created
+    local dump=`ls -l /tmp | grep noobaa_db_dump | awk '{ print $9 }'`
+    if [ ! -f "$dump" ]
+    then
+        echo_time "❌  db dump was not generated"
+        exit 1
+    fi
+
+    # Remove dump file
+    rm /tmp/$dump
+
+    # Generate db dump through diagnose API
+    echo_time "💬  Generaiting db dump through diagnose"
+    test_noobaa diagnose --db-dump --dir /tmp
+
+    # Check whether dump was create
+    local diagnose=`ls -l /tmp | grep noobaa_diagnostics | awk '{ print $9 }'`
+    dump=`ls -l /tmp | grep noobaa_db_dump | awk '{ print $9 }'`
+    if [ ! -f "$dump" ]
```

Comment:
Now it is not correct,  you are doing `ls -l` you will not get only the file name and you will also not get the path.

Reply:
Right, didn't want to change this actually, but committed it by mistake, anyway it's fixed.